src/pages/en/reference/adapter-reference.md

```diff
@@ -156,3 +156,31 @@ if(app.match(request)) {
 ```
 
 You can usually call `app.render(request)` without using `.match` because Astro handles 404s if you provide a `404.astro` file. Use `app.match(request)` if you want to handle 404s in a different way.
+
+## Allow installation via `astro add`
+
+[The `astro add` command](/en/reference/cli-reference/#astro-add) allows users to easily add integrations and adapters to their project. If you want _your_ adapter to be installable with this tool, **add `astro-adapter` to the `keywords` field in your `package.json`**:
+
+```json
+{
+  "name": "myadapter",
+  "keywords": ["astro-adapter"],
+  ...
+}
+```
+
+Once you [publish your adapter to npm](https://docs.npmjs.com/cli/v8/commands/npm-publish), running `astro add myadapter` should install and apply your adapter to the user's `astro.config` like so:
+
+```diff
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
++ import myadapter from 'myadapter';
+
+export default defineConfig({
++  adapter: myadapter(),
```

Comment:
We are using a specific pattern of camelCase and kebab-case in the SSR page and this pattern is also used in the Netlify Edge Functions adapter ('@astrojs/netlify/edge-functions'). I think it would be nice to use the same pattern everywhere in the docs, what do you think? This need to be changed in a few other places, I'm going to add the changes as suggestions.
```suggestion
+ import myAdapter from 'my-adapter';

export default defineConfig({
+  adapter: myAdapter(),
```

Reply:
That's unfortunate! :cry: This means we need to change the pages in where we use this pattern then, I can take care of this tomorrow, thanks for testing and finding this behavior!

Reply:
Oh boy, good to know! Tbh, I’d rather fix the underlying behavior before editing the docs. Hoping it’s easy to solve 😄 Hang tight!

Reply:
Just checking, but it seems all of our own adapters and integrations are all single words... so I guess we don't have this issue (of which case to use) when describing any of our own. It seems to only be an issue to document right now because we're trying to use a 2-word phrase "my adapter." 
 
Is there a one-word, generic term that fits so that we avoid the issue? (That isn't "adapter"?  😅 )

Do we provide any suggestions for people naming their own? Is anyone likely to run into a problem choosing a particular multi-word name?

Otherwise, LGTM when you both are happy with the generic placeholder name!

Reply:
@sarah11918 Think I'll just use `example` to fix it 🙃